fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -30,38 +31,64 @@ import androidx.compose.runtime.setValue
 import androidx.compose.runtime.snapshotFlow
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.layout.onGloballyPositioned
 import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
 import androidx.compose.ui.semantics.clearAndSetSemantics
 import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.role
 import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.DpOffset
 import androidx.compose.ui.unit.dp
+import mozilla.components.concept.engine.translate.Language
 import mozilla.components.concept.engine.translate.TranslationError
 import org.mozilla.fenix.R
 import org.mozilla.fenix.compose.BetaLabel
+import org.mozilla.fenix.compose.ContextualMenu
 import org.mozilla.fenix.compose.LinkText
 import org.mozilla.fenix.compose.LinkTextState
+import org.mozilla.fenix.compose.MenuItem
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
 import org.mozilla.fenix.compose.button.PrimaryButton
 import org.mozilla.fenix.compose.button.TertiaryButton
 import org.mozilla.fenix.compose.button.TextButton
 import org.mozilla.fenix.shopping.ui.ReviewQualityCheckInfoCard
 import org.mozilla.fenix.shopping.ui.ReviewQualityCheckInfoType
 import org.mozilla.fenix.theme.FirefoxTheme
+import java.util.Locale
+
+private val ICON_SIZE = 24.dp
 
 /**
  * Firefox Translations bottom sheet dialog.
+ *
+ * @param learnMoreUrl The learn more link for translations website.
+ * @param showFirstTimeTranslation Whether translations first flow should be shown.
+ * @param translationError The type of translation errors that can occur.
+ * @param translateFromLanguages Translation menu items to be shown in the translate from dropdown.
+ * @param translateToLanguages Translation menu items are to be shown in the translate to dropdown.
+ * @param onSettingClicked Invoked when the user clicks on the settings button.
+ * @param onLearnMoreClicked Invoked when the user clicks on the Learn More button.
+ * @param onTranslateButtonClick Invoked when the user clicks on the Translate button.
+ * @param onNotNowButtonClick Invoked when the user clicks on the Not Now button.
```

Comment:
```suggestion
 * @param onLearnMoreClicked Invoked when the user clicks on the "Learn More" button.
 * @param onTranslateButtonClick Invoked when the user clicks on the "Translate" button.
 * @param onNotNowButtonClick Invoked when the user clicks on the "Not Now" button.
```
Let's add quotations around the button label text so we can map it more easily to the exact string.

Reply:
Done